figures/scripts/figS6_UMAPs_seq-center.R

```diff
@@ -0,0 +1,119 @@
+# S. Spielman 2022
+# Script to make UMAPs exploring potential sequencing center batch effects
+
+# Libraries --------
+library(magrittr)
+
+# Define paths and files ---------------------
+dim_red_dir <- file.path(
+  "analyses",
+  "transcriptomic-dimension-reduction"
+)
+
+metadata_file <- file.path("data", "pbta-histologies.tsv")
+pal_file <- file.path("figures", "palettes", "broad_histology_cancer_group_palette.tsv")
+rsem_umap_file <- file.path(
+  dim_red_dir,
+  "results",
+  "rsem_stranded_log_umap_scores_aligned.tsv"
+)
+
+
+# Output:
+output_dir <- file.path("figures", "pdfs", "supp", "figS5", "panels")
```

Comment:
Assuming we are making a figure S6, as implied by the title.
```suggestion
output_dir <- file.path("figures", "pdfs", "supp", "figS6", "panels")
```

Reply:
S5 is definitely what it should _all_ say 😬 😝. This will go "away" in a notebook, either way. We can save potential figure exports for later.